vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj.filters

```diff
@@ -268,6 +268,8 @@
     <ClCompile Include="Modules\AppStateData.cpp">
       <Filter>Modules</Filter>
     </ClCompile>
+    <ClCompile Include="Modules\ClipboardModule.cpp" />
+    <ClCompile Include="Modules\ClipboardModule.cpp" />
```

Comment:
>Modules\ClipboardModule.cpp" [](start = 24, length = 28)

Duplication of line above?

Reply:
It is probably a bad merge. I will fix it.